src/sagemaker/workflow/pipeline.py

```diff
@@ -43,6 +43,59 @@
 from sagemaker.workflow.utilities import list_to_request
 
 
+def load(pipeline_arn: str):
+    """Loads a new Pipeline object with information from a pipeline in the Tioga backend.
+
+    The steps in the object will be left empty
+
+    Args:
+        pipeline_arn (str): pipeline arn for a specific pipeline in Tioga backend
+
+    Returns:
+        Pipeline object
+    """
+    pipelineEntity = Pipeline.sagemaker_session.sagemaker_client.describe_pipeline(pipeline_arn)
+    pipelineDefinition = pipelineEntity["PipelineDefinition"]
+    return Pipeline(pipelineEntity["PipelineName"], pipelineDefinition["Parameters"], [])
+
+
+def build_visual_dag(
+    pipeline_name: str, adjacency_list: List[Dict[str, any]], step_statuses: Dict[str, str]
+):
+    """Builds a Graphviz object that visualizes a pipeline/execution
+
+    Args:
+        pipeline_name (str): pipeline name for the visualized pipeline
+        adjacency_list (List[Dict[str, any]]): adjacency list for the visualized pipeline
+        step_statuses (Dict[str, str]): step statuses of the steps in an execution
+
+    Returns:
+        A Graphviz object
+    """
+    try:
+        import graphviz
+    except ImportError:
+        raise ImportError(
+            "Please import graphviz package to use this method. Call 'pip install graphviz' "
+            "and rerun command"
+        )
+
+    color = {"Succeeded": "green", "Failed": "red", "Executing": "blue", "Not Executed": "grey"}
+
+    G = graphviz.Digraph(pipeline_name, strict=True)
+
+    for step in adjacency_list:
+        parent = step["StepName"]
+        status = step_statuses[parent] if parent in step_statuses else "Not Executed"
+        G.node(parent, color=color[status], style="filled")
+        for child in step["OutBoundEdges"]:
+            child_name = child["nextStepName"]
+            edge = child["edgeLabel"]
+            G.edge(parent, child_name, label=edge)
+
```

Comment:
IIRC, the `edge` can be null/None right? then the edge in the graph would not print anything? or print out a "null"/"None"?

Reply:
The edge can be null/None and will not print anything out, so the edge will not have a label